extensions/wikia/AdEngine/js/wad/btRecLoader.js

```diff
@@ -14,40 +14,60 @@ define('ext.wikia.adEngine.wad.btRecLoader', [
 		placementsMap = {
 			TOP_LEADERBOARD: {
 				uid: '5b33d3584c-188',
-				style: 'margin: 10px 0;'
+				style: 'margin:10px 0; z-index:100;',
+				lazy: false
 			},
 			TOP_RIGHT_BOXAD: {
 				uid: '5b2d1649b2-188',
-				style: 'margin-bottom: 10px;'
+				style: 'margin-bottom:10px; z-index:100;',
+				lazy: false
+			},
+			BOTTOM_LEADERBOARD: {
+				uid: '5b8f13805d-188',
+				style: 'margin-bottom:23px; z-index:100;',
+				lazy: true
 			}
 		};
 
-	function markAdSlots(replace) {
+	function markAdSlots(replace, lazySlotName) {
 		Object
 			.keys(placementsMap)
 			.forEach(function (key) {
```

Comment:
We don't need to iterate when lazySlotName is defined. Am I right?

Reply:
What do you think about extracting separate method for `markAdSlot(slotName)` and reuse it in `markAdSlots`

Reply:
Good suggestions. Code refactored.